### ISSUE TITLE
Header button to guide page

### ISSUE THREAD

**Issue by jcconnol** [freeCodeCamp/freeCodeCamp#34484]
I don't know if this has been thought of or not but I have had a profile on the website for a while and have not known there were guides for pretty much everything. I would like to make a button in the header that takes the user to the guides page (https://guide.freecodecamp.org/).
I have read the contribution Guidelines and there is nothing about adding functionality.
I do not know if that would make the header bar look cluttered.
This would be my first contribution and so I would like to make the pull request and changes.

**Comment by T1mL3arn** [comment]
If my memory is good, some time ago fcc had a different interface and there was that button in the header. Will be nice to see "Guides" button in the header again. Currently there are only(?) two ways to get there: by a searchbar in the header or by "Get a hint" button from a challenge page.

**Comment by StephenHogsten** [comment]
@jcconnol are you still planning on submitting a PR?  This is always bugging me when I use the site

**Comment by RandellDawson** [comment]
@jcconnol, @jcconnol I do not believe they are going to add anything else to the top nav anytime soon.  They have moved the guide link to the footer in the latest version of code on the master branch and will be moving this change to production in the very near future.   I would let some other collaborators weigh in on this before creating a PR right now.  I will label this as 'discussing' for now until a core team member can reply.

**Comment by jcconnol** [comment]
@StephenHogsten I was working on it very slowly on and off but if it is something that will already be put into the master branch then I will leave it be.

**Comment by moT01** [comment]
I agree that the guide is hard to find - I would like to see a link in the nav, but I don't think that's gonna happen.
